kernel/src/expressions/mod.rs

```diff
@@ -15,6 +15,8 @@ use crate::DataType;
 mod column_names;
 mod scalars;
 
+pub(crate) mod single_row_transform;
```

Comment:
need to predicate including this on having an arrow dep

Reply:
commented above but just adding here: shouldn't be dependent on arrow since this is just `Schema` to `Expression`